templates/todo/projects/java-mongo-aca/.repo/bicep/infra/main.bicep

```diff
@@ -31,6 +31,12 @@ param webAppExists bool = false
 @description('Flag to use Azure API Management to mediate the calls between the Web frontend and the backend API')
 param useAPIM bool = false
 
+@description('Hostname suffix for container registry. Set when deploying to sovereign clouds')
+param containerRegistryHostSuffix string = 'azurecr.io'
```

Comment:
Same as above?  I'm assuming this to support OOTB support for gov cloud.  At the same time I feel like anyone adopting gov cloud is already used to making tweaks for their scenarios.

Reply:
`azurecr.io` is hardcoded in https://github.com/Azure/azure-dev/blob/08bf610064a73b9a068233fab150021b8bd074c9/templates/common/infra/bicep/core/host/container-app.bicep#L127 

Being able to pass this through the environment enables these templates to work in other clouds.